### PR TITLE
load tests instructions for minikube

### PR DESCRIPTION
**Pull request by nustiueudinastea**
For consistency, added instructions on how to perform load tests for k8s minikube. Related to this request: https://github.com/microservices-demo/microservices-demo/issues/207

### CODE REVIEW

File: docs/deployment/kubernetes-minikube.md [hunk 1]
```diff
@@ -45,13 +45,22 @@ kubectl get pods --namespace="sock-shop"
 
 Once the application is deployed, navigate to http://192.168.99.100:30001 to see the Sock Shop home page.
 
+### Run tests
+
+There is a separate load-test available to simulate user traffic to the application. For more information see [Load Test](#loadtest).
+This will send some traffic to the application, which will form the connection graph that you can view in Scope or Weave Cloud.
+
+```
+docker run --rm weaveworksdemos/load-test -d 5 http://192.168.99.100:30001 -c 3 -r 10
```
Comment (vlal-zz) [inline docs/deployment/kubernetes-minikube.md]: missing `-h` flag

File: docs/deployment/kubernetes-minikube.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ There is a separate load-test available to simulate user traffic to the applicat
 This will send some traffic to the application, which will form the connection graph that you can view in Scope or Weave Cloud.
 
 ```
-docker run --rm weaveworksdemos/load-test -d 5 http://192.168.99.100:30001 -c 3 -r 10
+docker run --rm weaveworksdemos/load-test -d 5 -h http://192.168.99.100:30001 -c 3 -r 10
```
Comment (pidster) [inline docs/deployment/kubernetes-minikube.md]: Have you tested this?

**Review by pidster — CHANGES_REQUESTED** [review]
Please verify that the load-test command has been tested as it is described.

**Comment by nustiueudinastea** [comment]
@pidster fixed the issue with the command.

**Review by pidster — APPROVED** [review]

**Pull request merged**